backend/server.py

```
@@ -2157,35 +2185,76 @@ async def ensure_bootstrap() -> None:
+            try:
+                await seed_default_providers()
+            except Exception as exc:  # noqa: BLE001
+                log.debug("seed_default_providers failed (non-fatal): %s", exc)
+
+            try:
+                await _sync_ollama_model()
+            except Exception as exc:  # noqa: BLE001
+                log.debug("Ollama model sync failed (non-fatal): %s", exc)
+
             _BOOTSTRAP_DONE = True
     except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)